examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,169 @@
+{
+"metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": [{
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
+            "ru": "Эбботт, Эдвин Эбботт",
+            "en": "Abbott, Edwin Abbott"
+        },
+    }, {
+        "name": "Bob Braggert",
+        "sort_as": "Braggert, Bob"
+    }],
+    "narrator": [
```

Comment:
For `narrator`, `illustrator` and `translator`, now that you only include `name`, a mort straightforward syntax could be used:

```
"narrator": ["Abe Narrator", "Beatrice Narrator"]
```